.eslintrc

```diff
@@ -0,0 +1,21 @@
+{
+  "ecmaFeatures": {
+    "jsx": true
+  },
+  "env": {
+    "browser": true,
+    "node": true
+  },
+  "rules": {
+    "quotes": 1,
+    "no-trailing-spaces": 1,
+    "strict": 1,
```

Comment:
Based on what I'm seeing [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode), it sounds like we should probably keep this. `"use strict";` turns a number of errors that would normally produce a warning (ex. trying to use `delete` on something that isn't deletable, overwriting specific es6 keywords that weren't implemented in es5, etc.) into an actual full-blown error.